fw/http.c

```diff
@@ -6756,10 +6759,14 @@ tfw_http_resp_process(TfwConn *conn, TfwStream *stream, struct sk_buff *skb)
 	 * @hmsib is not attached to the connection yet.
 	 */
 	r = tfw_http_resp_cache(hmresp);
-	if (unlikely(r != T_OK))
```

Comment:
Seems in line `6740:goto next_resp;` we must call `tfw_http_popreq` to dequeue req from fwd queue.

Reply:
I totally remove this code since frang always return error if we should drop connection